app/poros/game_party.rb

```diff
@@ -0,0 +1,17 @@
+class GameParty
+  attr_reader :id, :name, :date, :user_id, :number_of_games, :friends
+  def initialize(data)
+    @id = data[:id]
+    @name = data[:attributes][:name]
+    @date = data[:attributes][:date]
+    @user_id = data[:attributes][:user_id]
+    @number_of_games = data[:attributes][:number_of_games]
+    @friends = []
```

Comment:
You can call the add friends method here in the initialization, also I think we could initialize the friends objects within that method rather than calling it in the facade, thats all if we decide to automatically add friends to a gamenight